app/containers/ItunesContainer/saga.js

```diff
@@ -17,6 +18,27 @@ export function* getItunesData(action) {
   }
 }
 
+export function* getTrackData(action) {
+  const trackData = yield select(selectItunesData());
+  const trackCache = trackData.results?.(action.trackId);
```

Comment:
I think you meant to go inside the results array and find it. Results is an array right?

Reply:
inside array of results find track id for the corresponding action

Reply:
How?